frontend/das_haus/src/App.js

```diff
@@ -1,11 +1,29 @@
 import 'bootstrap/dist/css/bootstrap.css';
 import './App.css';
-import NavigationBar from './components/Navbar.js'
-
+import { RouterProvider, createBrowserRouter } from 'react-router-dom';
+import HomePage from './pages/Home'
+import RootLayout from './pages/Root';
+import AuthenticationPage from './pages/Auth';
+import DummyProtectedPage from './pages/Dummy'
+import {action as authAction} from './components/AuthForm'
+import {routeProtectionLoader} from './util/routeProtection'
 function App() {
-  return (
-    <NavigationBar/>
-  )
+
+  const router = createBrowserRouter([
+    {
+      path: '/',
+      element: <RootLayout />,
+      children: [
+        { path: '/', index: true, element: <HomePage /> },
+        {path: 'auth/', element: <AuthenticationPage/>, action: authAction, name:"auth"},
+        {path: 'dummy', element: <DummyProtectedPage/>, loader: routeProtectionLoader}
+
+
+
+      ]}])
+      return <RouterProvider router={router} />;
```

Comment:
remove whitespace here. 
You can leave the return statement on the last line, with a new line before the return.